### PR TITLE
Update LLVM

### PR DESCRIPTION
**Pull request by trilorez**
`llvm::cl::MiscFlags::CommaSeparated` no longer needs to be specified for `ListOption` as of [this patch](https://reviews.llvm.org/rG6edef1356921d9cad1a8cd6169207450741536a6)

### CODE REVIEW

**Review by teqdruid — APPROVED** [review]
LGTM! Thanks

File: include/circt/Dialect/MSFT/MSFTPasses.td [hunk 1]
```diff
@@ -23,7 +23,7 @@ def ExportTcl: Pass<"msft-export-tcl", "mlir::ModuleOp"> {
   let options = [
     ListOption<"tops", "tops", "std::string",
                "List of top modules to export Tcl for",
-	       "llvm::cl::ZeroOrMore, llvm::cl::MiscFlags::CommaSeparated">,
+	             "llvm::cl::ZeroOrMore,">,
```
Comment (mikeurbach) [inline include/circt/Dialect/MSFT/MSFTPasses.td:26]: nit: unrelated format change.

also: do you have a .td formatter you are using? i've wished for such a thing, so the community can standardize on tablegen format like how clang-format makes the c++ format unambiguous.

**Review by mikeurbach — APPROVED** [review]
I checked the upstream patch, and this looks like the right way to update downstreams.

File: include/circt/Dialect/MSFT/MSFTPasses.td (hunk 1 above)
Comment (trilorez) [reply include/circt/Dialect/MSFT/MSFTPasses.td]: I don't have a formatter, I just inferred that the original indentation was due to the longer line length. Also I didn't realize there was a tab in there, I've updated it to be consistent with the other lines, but I can change it to the original indentation if you prefer.

Looks like a td formatter is [being proposed](https://discourse.llvm.org/t/tablegen-formatter/60418).

Comment (mikeurbach) [reply include/circt/Dialect/MSFT/MSFTPasses.td]: Gotcha, removing the tab looks good to me.
